src/views/home/UserInput.js

```diff
@@ -30,7 +30,7 @@ export default class UserInput extends Component {
           height: '',
           width: '',
           filename: 'screenshot0.png',
-          userAgent: '',
+          useragent: '',
```

Comment:
The [api](https://github.com/csci4950tgt/api/blob/master/models/screenshot.go#L13) is expecting camelCase here. Changing the case will stop the backend from reading the property correctly.

Reply:
Ahhhh, so I changed this because the browser was expecting lowercase and was logging console errors because of this, I can play around with that and see about keeping the state camelcase but when rendering components rendering it in lowercase